api/datamart/migrations/20250613145435_create-active_calibrated_challenges-table.js

```diff
@@ -0,0 +1,17 @@
+const TABLE_NAME = 'active_calibrated_challenges';
+
+const up = async function (knex) {
+  await knex.schema.createTable(TABLE_NAME, function (table) {
+    table.string('challenge_id');
+    table.float('alpha');
+    table.float('delta');
+    table.string('scope');
```

Comment:
Manque un `index()` sur le scope !

```suggestion
    table.string('scope').index();
```